content/sips/sip-161.md

```diff
@@ -48,72 +50,155 @@ On the topic of scaling and decentralizing the Synthetix Protocol, the onboardin
+
+The CCC upon receiving the vouches, decides that it is a suitable candidate because of her previous involvement with the grantsDAO, they look at their budget and find that there is room to offer Sally a incentive package. Sally happily receives it and is on-boarded as a Core Contributor.
+
+_Compensation Review_
+
+Billy, an existing Core Contributor was on-boarded as a front-end engineer into the Synthetix Protocol to work on the dApps. Over the last 3 months, he has taken up Solidity development, writing up helper contracts to aggregate data for the dApps to consume easily.
+
+The Core Contributor Committee has noticed this and so on the next quarterly review, they decide to offer Billy a higher compensation package and also ask if Billy wants to focus more on core contract development.
+
+Billy who is astounded to hear this notices that the new compensation package does not match his expectations and decides to go back to the CCC with a higher offer. The CCC looks at their budget and decides that it can be provided and happily accepts.
+
+_Discretion in Budgeting_
+
+The Core Contributor Committee has received their quarterly budget, upon review it is enough to meet the stipend required for the existing Core Contributors. However, the Core Contributor Committee is planning to hold a Core Contributor Off-Site at Main-net New York as well as sponsor it and realize that they need more funding for this particular quarter.
+
+The Core Contributor Committee schedules a meeting with the Treasury Council in which they request more funding for partnerships and also ancillary cost. The Treasury Council obliges and schedules a transfer of funds to the Core Contributor Committee multi-sig.
+
+The CC's are happy as they have had a rough year and can finally meet their colleagues in person.
 
-_Core Contributor Engagement_ the Core Contributor Committee is in the best position to understand which resources are lacking within the Core Contributors and the budget capacity while the existing Core Contributors themselves understand which candidates would be most suitable for the role they are applying for, this vouching system ensures that both parties are responsible for what they know best.
+_Roadmap & Priorities_
 
-_Core Contributor Compensation_ the CCC have full discretion over the budget provided by the Treasury Council so they are able to ensure that the Core Contributors incentives are aligned with the Synthetix Protocol vision.
+There are 5 SIPs in total sitting in the **FEASIBILITY** and **APPROVED** categories in a particular week, when the CCC meet with the Spartan Council they decide that one of the particular approved SIPs is important for the Synthetix Stakers and is time-sensitive. They decide on making this SIP the highest priority and the CCC schedules this in.
```

Comment:
One thing we've struggled with in the past is that Approved SIPs sometimes sit there without an implementor CC assigned. This may be a problem of the past, but should we call out that the CCC will be responsible to make sure that Approved sips don't sit there for a while "unowned"?